src/listeners/event-message.js

```diff
@@ -0,0 +1,70 @@
+function handler(event, client) {
+  console.log('\n====================START HANDLE MESSAGE====================')
+  console.log('Resource Type:', event.resourceType)
+  // console.log('EVENT:', event)
+  switch (event.resourceType) {
+    case 'ConversationUpdate': {
+      client.emit('reply-message', event)
+      // handleReply(event)
+      break
+    }
+    case 'NewMessage': {
+      const threadType = event.resource.threadtype
+
+      if (threadType === 'topic') {
+        console.log('in topic')
+        client.emit('new-message', event)
+        // handleNewMessage(event)
+      } else if (threadType === 'streamofnotifications') {
+        console.log('in notificationstream')
+        client.emit('message-notification-stream', event)
+        // handleMessageNotification(event)
+      }
+      break
+    }
+    case 'MessageUpdate': {
+      client.emit('update-message', event)
+      // handleMessageUpdate(event)
+      break
+    }
+    default: {
+      console.log('Unknown resource type')
```

Comment:
Normally I would want this gone as well, but this would help identify new resource types in the future... I think this one should stay.

Reply:
I agree, leaving it in for now.